internal/reconcilerevent/cluster.go

```diff
@@ -34,3 +35,15 @@ func (s *PeerCreatedEvent) Record(recorder record.EventRecorder) {
 		"PeerCreated",
 		fmt.Sprintf("Created a new EtcdPeer with name '%s'", s.PeerName))
 }
+
+type MemberAddedEvent struct {
+	Object runtime.Object
+	Member *etcdclient.Member
+}
+
+func (s *MemberAddedEvent) Record(recorder record.EventRecorder) {
+	recorder.Event(s.Object,
+		"Normal",
+		"MemberAdded",
+		fmt.Sprintf("Added a new member with name %s", s.Member.Name))
```

Comment:
In `isAllMembersStable` it implies that etcd API doesn't report Member name  until that member has started and connected to the cluster.
In which case, won't this always be empty?

Reply:
Yes, will correct.